pyproject.toml

```diff
@@ -0,0 +1,105 @@
+[project]
+name = "ga4gh.vrs"
+authors = [
+    {name = "Larry Babb", email = "lbabb@broadinstitute.org"},
+    {name = "Reece Hart", email = "reecehart@gmail.com"},
+    {name = "Andreas Prlic", email = "andreas.prlic@gmail.com"},
+    {name = "Alex Wagner", email = "alex.wagner@nationwidechildrens.org"},
+]
+readme = "README.md"
+description = "GA4GH Variation Representation Specification (VRS) reference implementation"
+license = {file = "LICENSE"}
+classifiers = [
+    "Development Status :: 3 - Alpha",
+    "Intended Audience :: Developers",
+    "Intended Audience :: Healthcare Industry",
+    "Intended Audience :: Science/Research",
+    "License :: OSI Approved :: Apache Software License",
+    "Operating System :: OS Independent",
+    "Programming Language :: Python",
+    "Programming Language :: Python :: 3",
+    "Programming Language :: Python :: 3.8",
+    "Programming Language :: Python :: 3.9",
+    "Programming Language :: Python :: 3.10",
+    "Programming Language :: Python :: 3.11",
+    "Programming Language :: Python :: 3.12",
+    "Topic :: Scientific/Engineering :: Bio-Informatics",
+    "Topic :: Scientific/Engineering :: Medical Science Apps.",
+]
+keywords = [
+    "bioinformatics",
+    "ga4gh",
+    "genomics",
+    "hgvs",
+    "spdi",
+    "variation"
+]
+requires-python = ">=3.8"
+dynamic = ["version"]
+dependencies = [
+    "bioutils",
+    "canonicaljson",
+    "coloredlogs",
+    "numpy",
+    "pyyaml",
```

Comment:
I think we can remove these. I don't see canonicaljson, coloredlogs, or numpy being used anywhere and I think pyyaml belongs in dev and notebooks

```suggestion
```

Reply:
Yeah looks like a lot of this was removed in 64fee4c1a441e1623cda18734aa28c9137afb14b

Reply:
also, numpy????